package.json

```diff
@@ -120,18 +120,19 @@
     "website:start-preview": "yarn run ts-node ./scripts/website/compile --watch --bundle preview",
     "website:start-testRunner": "yarn run ts-node ./scripts/website/compile --watch --bundle testRunner",
     "website:start-server": "yarn run ts-node ./scripts/website/compile --watch --bundle server",
+    "website:start-server-test": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --bundle server",
```

Comment:
The `website:start-prod` script uses line 122 above this line for the server. Compare to the `website:start-server-prod` script below. The `website:start-server-prod` script doesn't seem to work. Using the `serve` package just makes the compiled files available. Adding the `--watch` flag like this command does works though.